packages/cmr-client/CMR.js

```diff
@@ -97,7 +99,7 @@ async function updateToken(cmrProvider, clientId, username, password) {
           username: username,
           password: password,
           client_id: clientId,
-          user_ip_address: await publicIp.v4().catch((_) => '127.0.0.1'),
+          user_ip_address: await publicIp.v4({ timeout: IP_TIMEOUT_MS }).catch((_) => '127.0.0.1'),
```

Comment:
This was causing issues when we were hitting this from the dashboard and would timeout. I lowered the timeout here and that fixed the issue.

Reply:
So this code just tries to get the users's IP address?  And lowering the timeout made it more successful?

Reply:
yes, I think it would timeout the API call and the dashboard didn't like that. This would take the full 5 seconds because it can't find a public IP in the VPC.